### DISCUSSION TITLE
Gallery for testing

### DISCUSSION THREAD

**Discussion by rodneyosodo** [nakujaproject/intern#67] [Show and tell]
This is a new thread where we post the things we have done. Mainly pictorial, video and links to written materials

**Comment by rodneyosodo** [comment]
# Avionics

Trying with the africastalking eris kit instead of stm32

![image](https://user-images.githubusercontent.com/28790446/110793207-6879b500-8285-11eb-942f-ea5c33b3e11b.png)

**Reply by rodneyosodo** [reply]
@oryankibandi and @Mike-Kimani Please add your avionics pictures and videos here

**Comment by rodneyosodo** [comment]
# Airframe

### 3D printing new nose cone

![image](https://user-images.githubusercontent.com/28790446/110793348-9101af00-8285-11eb-9893-076c883a6493.png)

### Testing the parachute mechanism

https://user-images.githubusercontent.com/28790446/110793482-b7bfe580-8285-11eb-928d-6836481c645c.mp4

**Reply by rodneyosodo** [reply]
@JeffMboya and @SammyOina Please add the airframe video you had here

**Reply by SammyOina** [reply]
# First Parachute Test

https://user-images.githubusercontent.com/44265300/110997391-618a9980-838e-11eb-8a9d-d498029a95bb.mp4

**Reply by SammyOina** [reply]
# Cross Parachute Test



https://user-images.githubusercontent.com/44265300/110997855-0ad18f80-838f-11eb-8310-431cb62f9a20.mp4

![IMG_20210312_162512](https://user-images.githubusercontent.com/44265300/110997972-35234d00-838f-11eb-8334-f32849e98661.jpg)

**Reply by rodneyosodo** [reply]
@SammyOina what about the failed parachute test

**Reply by kimanimichael** [reply]
https://user-images.githubusercontent.com/49106674/111130507-5733ef80-8588-11eb-8165-53d186c1ccc4.mp4

Parachute test video

**Reply by kimanimichael** [reply]
https://user-images.githubusercontent.com/49106674/111130582-716dcd80-8588-11eb-9d7c-48ef47b03dec.mp4

Parachute test with deployment initiated by a servo motor

**Reply by rodneyosodo** [reply]
Pressure analysis

![image](https://user-images.githubusercontent.com/28790446/111350285-00ffa300-8693-11eb-8245-7bfbab1cdd5b.png)

Simulation using traces to model airfrow

https://user-images.githubusercontent.com/28790446/111350605-55a31e00-8693-11eb-943f-d165a5330219.mp4

**Reply by shohei** [reply]
> Pressure analysis
> 
> 
> Simulation using traces to model airfrow
> Scene.mp4

@0x6f736f646f This is awesome! I'm wondering if I'm the only one that can't see the mp4. It's all black screen.

**Reply by rodneyosodo** [reply]
Let me upload another one

**Reply by rodneyosodo** [reply]
https://user-images.githubusercontent.com/28790446/111422608-00e9bc80-8700-11eb-86fb-cf621fe878c7.mp4

**Reply by rodneyosodo** [reply]
**Static analysis for locking mechanism**

https://user-images.githubusercontent.com/28790446/112021929-d904b980-8b42-11eb-8a69-5b6a787e1e0d.mp4

**Reply by rodneyosodo** [reply]
@Mike-Kimani please upload test for Friday 19th March 20201

**Reply by kimanimichael** [reply]
https://user-images.githubusercontent.com/49106674/112299359-c60bf980-8ca8-11eb-9aba-357091f88c6a.mp4

Successful parachute test from last Friday

**Reply by rodneyosodo** [reply]
@Mike-Kimani Please upload test for Wednesday 24th March

**Comment by rodneyosodo** [comment]
# Propulsion

**Reply by rodneyosodo** [reply]
@SammyOina Please add the propulsion images and videos here

**Reply by SammyOina** [reply]
## N2 Nozzle CFD
![n2-test-nozzle](https://user-images.githubusercontent.com/44265300/111199604-e285a300-85d1-11eb-9f0e-1472375869c0.JPG)

**Reply by JeffMboya** [reply]
N-1 test stand

![WhatsApp Image 2021-03-17 at 12 07 07 (1)](https://user-images.githubusercontent.com/44696487/111740825-76879100-8896-11eb-8b65-725c7723552b.jpeg)
![WhatsApp Image 2021-03-17 at 11 58 32 (1)](https://user-images.githubusercontent.com/44696487/111740830-78515480-8896-11eb-88db-5ed468fb8ed6.jpeg)
![WhatsApp Image 2021-03-17 at 12 09 26](https://user-images.githubusercontent.com/44696487/111740831-78e9eb00-8896-11eb-966a-1180f22a0303.jpeg)

**Reply by rodneyosodo** [reply]
@shohei and @JeffMboya  Please upload the motor test

**Reply by shohei** [reply]
https://user-images.githubusercontent.com/855816/112037288-a4006300-8b52-11eb-9cd3-e1d222dcda5e.mp4
